index.html

```diff
@@ -243,6 +243,32 @@ <h1>
                 </div>
             </div>
         </section>
+        <section class="contact-me">
+
```

Comment:
- [ ] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follow HTML best practices.

Reply:
Thank you so much. Let me do the adjustments ASAP 👍